config/Config.cfc

```diff
@@ -16,6 +16,7 @@ component {
 		settings.features.restTokenAuth.enabled = true;
 
 		settings.features.dataApiQueue = settings.features.dataApiQueue ?: { enabled=true };
+		settings.features.dataApiQuotesForNumerics = settings.features.dataApiQuotesForNumerics ?: { enabled=false };
```

Comment:
I think this feature name is a little unclear. Could we use `dataApiUseNullForNumerics` and set to _enabled_ by default instead? (and then update the logic that uses it)